cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/common/DataprocUtils.java

```diff
@@ -229,6 +234,20 @@ public static String getSystemProjectId() {
     }
   }
 
+  /**
+   * Emit a dataproc metric.
+   **/
+  public static void emitMetric(MetricsContext metricsContext, ProgramRunInfo programRunInfo,
+                                StatusCode.Code statusCode, String  metricName) {
+    Map<String, String> tags = ImmutableMap.of(Constants.Metrics.Tag.NAMESPACE, programRunInfo.getNamespace(),
```

Comment:
Shouldn't require constructing the full tag from the provisioner. Otherwise all provisioner implementation has to do this.

Reply:
fixed